scripts/aggregate.py

```diff
@@ -293,7 +293,7 @@ def main():
                                                         all_resolved,
                                                         len(failures['Unresolved']))
     unresolved_markdown = additional_failures_to_markdown("Unresolved",
```

Comment:
Can you move this down below where we write the markdown so it's clear that it isn't included in the args.output_markdown file?